docs/core/metrics_datadog.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Datadog
+description: Core utility
+---
+<!-- markdownlint-disable MD013 -->
+Datadog provider creates custom metrics by flushing metrics to standard output and exporting metrics using [Datadog Forwarder](https://docs.datadoghq.com/logs/guide/forwarder/?tab=cloudformation){target="_blank" rel="nofollow"} or flushing metrics to [Datadog extension](https://docs.datadoghq.com/serverless/installation/python/?tab=datadogcli){target="_blank" rel="nofollow"} using Datadog SDK.
```

Comment:
Add Datadog Extension first, then Forwarder so we're consistent with the best practice (and the rest of the docs which does this already).